src/app/containers/model/model.container.html

```diff
@@ -9,43 +9,63 @@
                             <mat-form-field floatLabel="always" class="form__field form__field--wide">
                                 <mat-label>Object Type</mat-label>
                                 <mat-select formControlName="objectType">
-                                    <mat-option *ngFor="let type of objectTypes" [value]="type" [disabled]="type === 'geometry file'">
-                                        {{type === 'plate' ? 'plate (one sided)' : type}}
-                                    </mat-option>
+                                    <ng-container *ngFor="let type of objectTypes">
+                                        <mat-option *ngIf="type !== 'geometry file'" [value]="type" >
+                                            {{type === 'plate' ? 'plate (one sided)' : type}}
+                                        </mat-option>
+                                    </ng-container>
+                                    <mat-optgroup label="geometry files (.wrl format):">
```

Comment:
When first opening up the dropdown menu I didn't realize there were other options below that I could scroll to. It stopped at SORCE.
Some ideas...

- Is there a way to force all of them to show up since we don't have that many?
- Make the scroll wheel permanently show up? (On my browser it auto-hides, which made this a bit tricky to know it was a scrollable field)
- Do a sub-field. So, when you go to the (currently gray) geometry files label, it breaks out to the right into another field selector just for the geometry files section.

Reply:
It should be full height on most screens now